[BUG] Quickstart Doesnt Compile with SRF Conda Package

**Describe the bug**
Building the QSG using the `./compile.sh` script fails when using the SRF conda package

**Steps/Code to reproduce bug**
If you run the following:

```
mamba install srf
cd docs/quickstart
./compile.sh
```

You will get the following error:

```
-- [quickstart.rapids.find.package] Found srf: ${CONDA_PREFIX}/lib/cmake/srf/srf-config.cmake (found version "22.6.0")
-- [quickstart] Found Python3: ${CONDA_PREFIX}/bin/python3.8 (found version "3.8.13") found components: Development Interpreter Development.Module Development.Embed
-- [quickstart] Found PythonInterp: ${CONDA_PREFIX}/bin/python (found version "3.8.13")
-- [quickstart] Found Cython: ${CONDA_PREFIX}/bin/cython
CMake Error at ${CONDA_PREFIX}/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find PythonLibs (missing: PYTHON_INCLUDE_DIRS)
Call Stack (most recent call first):
  ${CONDA_PREFIX}/share/cmake-3.22/Modules/FindPackageHandleStandardArgs.cmake:594 (_FPHSA_FAILURE_MESSAGE)
  ${CONDA_PREFIX}/share/cmake-3.22/Modules/FindPythonLibs.cmake:310 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  ${CONDA_PREFIX}/lib/python3.8/site-packages/skbuild/resources/cmake/UseCython.cmake:105 (find_package)
  ${CONDA_PREFIX}/lib/python3.8/site-packages/skbuild/resources/cmake/FindCython.cmake:82 (include)
  ./cmake/python_module_tools.cmake:82 (find_package)
  CMakeLists.txt:45 (include)
```

--- a/docs/quickstart/CMakeLists.txt
+++ b/docs/quickstart/CMakeLists.txt
@@ -32,6 +32,10 @@ project(srf-quickstart
 # Ensure CPM is initialized
 rapids_cpm_init()
 
+# Set the option prefix to match the outer project before including. Must be before find_package(srf)
+set(OPTION_PREFIX "SRF")
+include(python_module_tools)
+
 rapids_find_package(srf REQUIRED)
 rapids_find_package(CUDAToolkit REQUIRED)
 
@@ -39,10 +43,6 @@ rapids_find_package(CUDAToolkit REQUIRED)
 if("${CMAKE_EXECUTABLE_SUFFIX}" STREQUAL "")
   set(CMAKE_EXECUTABLE_SUFFIX ".x")
 endif()
-
-# Set the option prefix to match the outer project before including
-set(OPTION_PREFIX "SRF")
-include(python_module_tools)
 
 set(CMAKE_POSITION_INDEPENDENT_CODE TRUE)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)

